pandas/core/algorithms.py

```diff
@@ -786,16 +787,84 @@ def duplicated(values, keep='first'):
           occurrence.
         - ``last`` : Mark duplicates as ``True`` except for the last
           occurrence.
-        - False : Mark all duplicates as ``True``.
+        - False : Mark all duplicates as ``True``. This option is not
+          compatible with ``return_index`` or ``return_inverse``.
+    return_index : boolean, default False
```

Comment:
pls pls let's keep the scope to the original. just *return_inverse*. If you want to do more things, make them PR's on top.